bin/propolis-server/src/lib/migrate/preamble.rs

```diff
@@ -20,26 +22,68 @@ impl Preamble {
         Preamble { instance_spec, blobs: Vec::new() }
     }
 
-    /// Checks to see if the serialized spec in this preamble is compatible with
-    /// the supplied `other_spec`.
+    /// Given the instance spec in this preamble (transmitted from the migration
+    /// source), replace any Crucible and viona backends with their
     ///
     /// This check runs on the destination.
-    pub fn check_compatibility(
+    pub fn get_amended_spec(
```

Comment:
nit, take it or leave it: this actually amends the spec, it doesn't just return an already existing amended spec...so maybe this should just be called `amend_spec`?

Reply:
Fixed in 7db1183.